README.md

```diff
@@ -51,6 +51,132 @@ After this, you will have a directory containing files used for creating a
 custom Jupyter widget. You will now be able to easily package and distribute
 your custom Jupyter widget.
 
+## Files Generated by cookiecutter
+Jupyter widgets are made of two parts:
+- Python package for the code executed in kernel as the back-end of the widget
+- Javascript NPM package for the front-end of the widget running in browser
```

Comment:
Javascript ~NPM~ package ...

(it's just a JavaScript package, it doesn't necessarily get published to NPM or anything)